### ISSUE TITLE
multiple monitor setup

### ISSUE THREAD

**Issue by binarygeek119** [YARC-Official/YARG#491]
i have 3 monitors. it be nice if i could pick which display to play on.

also splitting each player to there own monitor would be cool aswell.

**Comment by EliteAsian123** [comment]
Yes, this is already planned and is already being worked on to an extent. Thanks for the suggestion!

**Comment by NicolasPL64** [comment]
Is there any news on this feature?

**Comment by AARP41298** [comment]
ward
